pkg/appconsts/consensus_consts.go

```diff
@@ -3,11 +3,9 @@ package appconsts
 import "time"
 
 const (
-	TimeoutPropose = time.Second * 10
-	TimeoutCommit  = time.Second * 11
 	// GoalBlockTime is the target time interval between blocks. Since the block
 	// interval isn't enforced at consensus, the real block interval isn't
 	// guaranteed to exactly match GoalBlockTime. GoalBlockTime is currently targeted
 	// through static timeouts (i.e. TimeoutPropose, TimeoutCommit).
-	GoalBlockTime = time.Second * 15
+	GoalBlockTime = time.Second * 6
```

Comment:
we can technically update this, as it is only ever updated in the genesis state, but it correctly causes the app hash tests to fail so its staying 15s for now

Reply:
tbc, this change has been reverted

Reply:
yea +1 on not modifying this rn